stdlib/2and3/numbers.pyi

```diff
@@ -22,42 +24,42 @@ class Complex(Number):
         def __nonzero__(self) -> bool: ...
     @property
     @abstractmethod
-    def real(self): ...
+    def real(self) -> Any: ...
     @property
     @abstractmethod
-    def imag(self): ...
+    def imag(self) -> Any: ...
     @abstractmethod
-    def __add__(self, other): ...
+    def __add__(self: _T, other: Any) -> _T: ...
```

Comment:
This isn't necessarily true; many concrete number classes return a different type from `__add__`. For example, int + float gives you a float. Unfortunately most of the methods on Number and Real probably have to just return Any.

Reply:
Yeah, that is unfortunate. I've removed all use of _T in this file and switched everything to Any.